packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/classes_table_diff.dart

```diff
@@ -127,6 +143,41 @@ class _InstanceColumn extends ColumnData<DiffClassStats> {
 
   @override
   bool get numeric => true;
+
+  @override
+  Widget? build(
+    BuildContext context,
+    DiffClassStats data, {
+    bool isRowSelected = false,
+    VoidCallback? onPressed,
+  }) {
+    if (!FeatureFlags.evalAndBrowse) return null;
+    final objects = _instances(data);
+    final theHeap = heap;
+
+    // There is no detailed information, if [dataPart] equals to [_DataPart.delta].
+    if (objects is! ObjectSet || theHeap == null) return null;
+
+    final theme = Theme.of(context);
+    final showMenu = isRowSelected;
+
+    final sampleObtainer =
+        showMenu ? HeapClassSampler(objects, theHeap, data.heapClass) : null;
+
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.end,
+      children: [
+        InstanceSetButton(
+          textStyle:
+              isRowSelected ? theme.selectedTextStyle : theme.regularTextStyle,
+          count: getValue(data),
+          gaContext: gac.MemoryAreas.snapshotSingle,
+          sampleObtainer: sampleObtainer,
+          showMenu: showMenu,
+        ),
+      ],
+    );
```

Comment:
this is duplicated in classes_table_single. Let's put into a helper widget so we don't have to define all this twice